src/app/core/layers/layer.js

```diff
@@ -20,6 +20,8 @@ const deprecate             = require('util-deprecate');
 
 // Base Class of all Layer
 function Layer(config={}, options={}) {
+  //DEV CODE
+  config.download_pdf = true;
```

Comment:
@volterra79 double-check

Reply:
@Raruto ok. Need to wait admin implementation